src/modules/dashboard/components/ColonyHome/ColonySubscription/ColonySubscription.css

```diff
@@ -1,19 +1,36 @@
 .main {
   composes: flexContainerRow flexAlignCenter from '~styles/layout.css';
-  margin-left: 2px;
+  margin-top: -4px;
```

Comment:
I really dislike negative margin. Why do we need it here? Can't we fix it any other way?

Reply:
Please leave a note if using negative margins explaining why